tests/test_nn/test_representation.py

```diff
@@ -207,9 +184,58 @@ def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMa
         return kwargs
 
 
+@unittest.skipIf(error is not None, "Need to install `torch_geometric`")
+class IgnoreRelationTypePyGRepresentationTests(cases.TriplesFactoryRepresentationTestCase):
+    """Test for Pytorch Geometric representations using uni-relational message passing layers."""
+
+    cls = pykeen.nn.pyg.IgnoreRelationTypePyGRepresentation
+    embedding_dim: int = 3
+    kwargs = dict(
+        base_kwargs=dict(shape=(embedding_dim,)),
+        layers=["gcn"] * 2,
+        layers_kwargs=dict(in_channels=embedding_dim, out_channels=embedding_dim),
+    )
+
+
+@unittest.skipIf(error is not None, "Need to install `torch_geometric`")
+class CategoricalRelationTypePyGRepresentationTests(cases.TriplesFactoryRepresentationTestCase):
+    """Test for Pytorch Geometric representations using categorical message passing layers."""
+
+    cls = pykeen.nn.pyg.CategoricalRelationTypePyGRepresentation
+    embedding_dim: int = 3
+    kwargs = dict(
+        base_kwargs=dict(shape=(embedding_dim,)),
+        layers=["rgcn"],
+        layers_kwargs=dict(
+            in_channels=embedding_dim,
+            out_channels=embedding_dim,
+            num_bases=2,
+            num_relations=cases.TriplesFactoryRepresentationTestCase.num_relations,
+        ),
+    )
+
+
+@unittest.skipIf(error is not None, "Need to install `torch_geometric`")
+class FeaturizedRelationTypePyGRepresentationTests(cases.TriplesFactoryRepresentationTestCase):
+    """Test for Pytorch Geometric representations using categorical message passing layers."""
+
+    cls = pykeen.nn.pyg.FeaturizedRelationTypePyGRepresentation
+    embedding_dim: int = 3
+    kwargs = dict(
+        base_kwargs=dict(shape=(embedding_dim,)),
+        layers=["gat"],
+        layers_kwargs=dict(
+            in_channels=embedding_dim,
+            out_channels=embedding_dim,
+            num_bases=2,
+            num_relations=cases.TriplesFactoryRepresentationTestCase.num_relations,
+        ),
```

Comment:
Vanilla `gat` doesn't have relations and `num_bases` arg, did you refer to `rgat` which [has](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.RGATConv) one?

Reply:
I think it was a copy-paste mistake from the test above 😅